testrail/run.py

```diff
@@ -22,6 +23,22 @@ def blocked_count(self):
         return self._content.get('blocked_count')
 
     @property
+    def case_ids(self):
```

Comment:
method name should be 'cases' and should return a list of Case objects.  

The setter would accept a list of Case objects and store the ids.